src/main/resources/assets/tconstruct/lang/en_us.json

```diff
@@ -628,16 +628,14 @@
     "item.tconstruct.flint_shuriken": "Flint Shuriken",
     "item.tconstruct.flint_shuriken.tooltip": "A throwable weapon with high knockback",
 
-    "item.tconstruct.green_slime_sling": "Slimesling",
+    "item.tconstruct.green_slime_sling": "Earth Slimesling",
     "item.tconstruct.green_slime_sling.tooltip": "Charge up, aim low, get flinging!\nUse Slime Boots if you value your life!",
-    "item.tconstruct.blue_slime_sling": "Slimesling",
-    "item.tconstruct.blue_slime_sling.tooltip": "Charge up, aim low, get flinging!\nUse Slime Boots if you value your life!",
-    "item.tconstruct.purple_slime_sling": "Slimesling",
-    "item.tconstruct.purple_slime_sling.tooltip": "Charge up, aim low, get flinging!\nUse Slime Boots if you value your life!",
-    "item.tconstruct.blood_slime_sling": "Congealed Slimesling",
-    "item.tconstruct.blood_slime_sling.tooltip": "Charge up, aim low, get flinging!\nUse Slime Boots if you value your life!",
-    "item.tconstruct.magma_slime_sling": "Slimesling",
-    "item.tconstruct.magma_slime_sling.tooltip": "Charge up, aim low, get flinging!\nUse Slime Boots if you value your life!",
+    "item.tconstruct.blue_slime_sling": "Sky Slimesling",
+    "item.tconstruct.blue_slime_sling.tooltip": "Charge up, let go, get back in the air!\nUse Slime Boots if you value your life!",
+    "item.tconstruct.purple_slime_sling": "Ender Slimesling",
+    "item.tconstruct.purple_slime_sling.tooltip": "Charge up, point it at a wall, come out the other side!",
+    "item.tconstruct.magma_slime_sling": "Magma Slimesling",
+    "item.tconstruct.magma_slime_sling.tooltip": "Charge up, point at a creature, get it out of my face!",
```

Comment:
This sentence reads weirdly, the switch from second person of the rest of the tooltips to first person here is awkward. Something like the following would be more consistent:
> Charge up, point at a creature, get it out of your face!

Though we could also phrase it a bit less aggressively:
> Charge up, point at a creature, and send it flying!